src/extension/common/protocol.re

```diff
@@ -1,8 +1,5 @@
 open Common;
-
-type result('a, 'e) =
-  | Ok('a)
-  | Error('e);
+open Result;
```

Comment:
No need for the open since all the `Result` namespace is used.

Reply:
Yeah IDK why that's there :3

Reply:
👍